### PR TITLE
Add one signal API calls to news and event pages

### PR DESCRIPTION
**Pull request by djimenezmsu**

### CODE REVIEW

**Comment by djimenezmsu** [comment]
You can view the notification calls [here](https://dashboard.onesignal.com/apps/4f2a57fd-aca7-459c-b63d-9f90f5498cf6/notifications)

File: src/app/actions/create-event.ts [hunk 1]
```diff
@@ -62,6 +64,13 @@ export async function createEvent(prevState: EventActionState, formData: FormDat
         } as EventActionState
     }
 
+    // send notification
```
Comment (clindsey4) [inline src/app/actions/create-event.ts:67]: 🗿🗿🗿

File: src/lib/onesignal/index.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        };
+        notification.contents = {
+            en: content
+        };
+        notification.included_segments = ["All"]; // we want this to go out to everyone
+        
+    try {
+        let notifResponse = await client.createNotification(notification);
+
+        if (notifResponse.errors) {
```
Comment (clindsey4) [inline src/lib/onesignal/index.ts:36]: 🗿👍

**Review by clindsey4 — APPROVED** [review]
😀😀😁

**Pull request merged**